platforms/shared/configuration/roles/helm_lint/tasks/main.yaml

```diff
@@ -14,10 +14,11 @@
 
 # create temp test directory
 - name: "Check or create test directory"
-  file:
+  include_role:
+    name: "../../shared/configuration/roles/check/setup"
```

Comment:
role name can be only check/setup ; they are in the same dir

Reply:
yes even i thought so but looks like it takes the reference from the playbook which ran it

Reply:
ok, then this is fine.